docs/faq.rst

```diff
@@ -1,28 +1,51 @@
 Frequently Asked Questions
 **************************
 
-Installation issues
+General Questions
 ===================
 
-Bambi not working on Windows
+What is the difference between Bambi and PyMC?
 ----------------------------
+    * Bambi is a regression library built on top of PyMC. It provides a simple 
+        interface for specifying Bayesian models, and allows for easy inference using MCMC or 
+        variational inference.
+    * PyMC is a library for Bayesian modelling, and is the backend used by Bambi. It is a very
+            powerful library, but can be challenging to use for beginners. Bambi provides a simple
+            interface for specifying models, and allows for easy inference via MCMC or variational
+            inference using PyMC.
 
-If Bambi does not work on Windows you can...
 
-* First
-* Second
-
-And finally, see `this issue <https://github.com/bambinos/bambi/issues/389>`_.
+Inference Questions
+====================
 
+What sampling methods are available?
+----------------------------
+The sampler used is automatically selected given the type of variables used in the model.
+For inference, Bambi supports both MCMC and variational inference. MCMC is the default, but you can specify variational inference by passing `inference_method='vi'` to `Model.fit()`.
+Bambi also supports multiple backends for MCMC, including NumPyro, and BlackJax
+(see API for "fit" method for more details [here](https://bambinos.github.io/bambi/api_reference.html)).
 
-Bambi not working on Linux
+Can infernece in Bambi be sped up using GPUs/TPUs?
 ----------------------------
+Yes, Bambi supports inference on GPUs and TPUs using the numpyro and blackjax backends. 
+See the API for "fit" method for more details 
+[here](https://bambinos.github.io/bambi/api_reference.html).
 
-* First
-* Second
+Model Specification Questions
+====================
 
+My data has a non-normal distributions, can I still use Bambi?
+----------------------------
+Yes, Bambi supports a wide range of distributions which can be specified using the "family"
+argument to the "Model". You can find examples of how to specify these distributions 
+in the [Bambi examples](https://bambinos.github.io/bambi/examples.html).
 
-Miscellaneous how to
-====================
+How do I find out what priors are available?
+----------------------------
+You can find a list of available priors in the [Bambi Models API](https://bambinos.github.io/bambi/api_reference.html#module-bambi.models) under the "family" argument.
```

Comment:
The family argument tells you what are the available families. But for priors, you can use any PyMC distribution that is valid as a prior.

Reply:
Updated as of 67f52fb6df